classes/autotools.oeclass

```diff
@@ -48,7 +48,22 @@ autotools_configure[emit] = "do_configure"
 do_configure () {
     do_configure_autotools
 }
+
+AUTOCONFDIR = ""
+S_AUTOCONFDIR = ""
```

Comment:
A package like autoconf have 2x config.{guess,sub} files.
This adds support for overwriting config.* in two seperate dirs...
Maybe one day a package have 3 locations for config.* files. Maybe this could be done differently... Any ideas?